nec_tables.py

```diff
@@ -438,4 +438,456 @@
                  5000: {'Cu': '700', 'Al': '1200'},
                  6000: {'Cu': '800', 'Al': '1200'}}
 
-conduit_type = []
+# Percent of Cross Section of conduit for Conductors and Cables from NEC 2017 Chapter 9, Table 1
+# (XX,YY)  :: XX=conductor/cable count, YY=conduit_xsection max fill
```

Comment:
I think you could simplify your 'legends' by just putting `(conductor/cable count, conduit_xsection maxfill)`

Reply:
Simplified all legends.